code/drasil-printers/lib/Language/Drasil/HTML/Print.hs

```diff
@@ -337,7 +337,7 @@ pItem (Nested s l) = vcat [pSpec s, makeList l]
 -----------------------------------------------------------------
 -- | Renders figures in HTML.
 makeFigure :: Doc -> Doc -> Doc -> L.MaxWidthPercent -> Doc
-makeFigure r c f wp = refwrap r (image f c wp)
+makeFigure r c f wp = refwrap r (image f (if c == mempty then Nothing else Just c) wp)
```

Comment:
As I said in a previous commit, `c` should not be of type `Doc` here -- the information of whether it is empty or not should come from the callee instead.